src/featureSlice/population/populationSlice.js

```diff
@@ -0,0 +1,65 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const initialState = {
+  Provinces: [],
+  state: 0,
+  population: 0,
+  isLoading: false,
+};
+
+const url = 'https://api.census.gov/data/2019/pep/charagegroups?get=NAME,POP&for=state:*';
+const countyurl = 'https://api.census.gov/data/2019/pep/charagegroups?get=NAME,POP&for=county:*&in=state:';
+
+export const getForAllProvinces = createAsyncThunk('population/getForAllProvinces',
+  async (thunkAPI) => {
+    try {
+      const resp = await axios(url);
+      // console.log(resp.data);
+      resp.data.shift();
+      const s = resp.data.map((a) => ({
+        Id: a[2],
+        State: a[0],
+        Population: a[1],
+        Counties: [],
+      })).sort((a, b) => (parseInt(a.Population, 10) > parseInt(b.Population, 10) ? -1 : 0));
+      // return resp.data;
+      // console.log(s);
```

Comment:
- [ ] Kindly remove commented code to keep the flow of your code.

Reply:
I am so sorry, I missed the important thing.

Reply:
You did great minor, issues

Reply:
@Wes-Isaac I removed the commented code